File: Javascript/7KYU/Ifs.js
var noIfsNoButs = function (a,b) {
  switch(true) {
    case (a==b): return a.toString() + " is equal to " + b.toString(); break;
    case (a>b): return a.toString() + " is greater than " + b.toString(); break;
    case (a<b): return a.toString() + " is smaller than " + b.toString(); break;
  }
}

File: Python/7KYU/Ifs.py
def no_ifs_no_buts(a, b):
    return {
        a == b: str(a) + " is equal to " + str(b),
        a < b: str(a) + " is smaller than " + str(b),
        a > b: str(a) + " is greater than " + str(b),
    }[True]